wikipendium/wiki/views.py

```diff
@@ -107,7 +109,15 @@ def add_tag_to_article(request, slug):
     if 'tag' not in request.POST:
         return HttpResponseBadRequest()
 
-    tag = slugify(request.POST['tag'])
+    slug = original_slug = slugify(request.POST['tag'])
```

Comment:
Why don't we have to keep the original `request.POST['tag']` before we slugify it? Won't this turn `økonomi` to `konomi`?